web/frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@viamrobotics/remote-control",
-  "version": "0.2.15",
+  "version": "0.2.16",
```

Comment:
@michaellee1019 - lmk if I should hold off on bumping this - not sure there's a release schedule or we release as changes as needed

Reply:
No specific release schedule, whenever you are ready. Before merging make sure that someone else hasn't changed the version as well.

After changing this version, go ahead and upgrade on App side, test to make sure it still works within App, and then PR it :)